app/assets/stylesheets/petitions/views/_shared.scss

```diff
@@ -91,6 +91,10 @@
   }
 }
 
+.petition-item.petition-rejected {
```

Comment:
Nice variation combo. Leave it as is but in future, a variation should have the base in the name so something like `.petition-item-rejected` would be better